pkg/app/launcher/cmd/launcher/launcher.go

```diff
@@ -227,6 +233,23 @@ func (l *launcher) run(ctx context.Context, input cli.Input) error {
 		if l.gitSSHKeyFile != "" {
 			options = append(options, git.WithGitEnv(fmt.Sprintf("GIT_SSH_COMMAND=ssh -i %s -o StrictHostKeyChecking=no -F /dev/null", l.gitSSHKeyFile)))
 		}
+		if l.gitSSHKeyData != "" {
+			decodedKey, err := base64.StdEncoding.DecodeString(l.gitSSHKeyData)
+			if err != nil {
+				return fmt.Errorf("failed to decode SSH key data, (%w)", err)
+			}
+			tmpKeyFile, err := os.CreateTemp("", "git-ssh-key-data")
+			if err != nil {
+				return fmt.Errorf("failed to create a temp file for SSH key data (%w)", err)
+			}
+			if _, err = tmpKeyFile.WriteString(string(decodedKey)); err != nil {
```

Comment:
nits.
We can use `Write` method as belows

```suggestion
			if _, err = tmpKeyFile.Write(decodedKey); err != nil {
```

Reply:
@Warashi 

thank you, I fixed it here https://github.com/pipe-cd/pipecd/pull/5258/commits/1c6e430f6ae1c3c35a72f723b7d262198ca0775b.

It's a legacy code and I missed.